From 357bea890354acda52aa6dfaec7fa232fa0b8208 Mon Sep 17 00:00:00 2001
From: Angelo Compagnucci <angelo.compagnucci@gmail.com>
Date: Tue, 23 May 2017 11:19:58 +0200
Subject: [PATCH] Remove unit-tests from mono compilation

This patch fixes compiling errors with unit-tests under linux.

Signed-off-by: Angelo Compagnucci <angelo.compagnucci@gmail.com>
---
 mono/Makefile.am | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/mono/Makefile.am b/mono/Makefile.am
index 8c9c2cb..7af36ec 100644
--- a/mono/Makefile.am
+++ b/mono/Makefile.am
@@ -34,7 +34,7 @@ monotouch-do-clean:
          (cd $$subdir && $(MAKE) $(AM_MAKEFLAGS) $$target); \
     done;
 else
-SUBDIRS = $(btls_dirs) arch utils io-layer cil metadata $(sgen_dirs) mini dis tests unit-tests benchmark profiler
+SUBDIRS = $(btls_dirs) arch utils io-layer cil metadata $(sgen_dirs) mini dis tests benchmark profiler
 endif
 endif
-DIST_SUBDIRS = btls arch utils io-layer cil metadata $(sgen_dirs) mini dis tests unit-tests benchmark profiler
+DIST_SUBDIRS = btls arch utils io-layer cil metadata $(sgen_dirs) mini dis tests benchmark profiler
-- 
2.7.4

